### PR TITLE
[homework #4] - CSS by Agustin Balbuena

### PR DESCRIPTION
**Pull request by agustinbalbuena**

### CODE REVIEW

File: homeworks/agustin-balbuena/04-css/index.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+          </li>
+        </ul>
+        <div class="profile-btns">
+          <a href="#"
+            >Download CV
+            <img src="media/icons/arrowDownload.svg" alt="download cv" />
+          </a>
+          <a href="#">Contact me</a>
+        </div>
+      </aside>
```
Comment (ilya-ischenko) [inline homeworks/agustin-balbuena/04-css/index.html:90]: Is does not looks like aside content

File: homeworks/agustin-balbuena/04-css/styles/layout.css [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+  border-radius: 50%;
+}
+.project-list a img {
+  width: 40px;
+  height: 40px;
+}
+.project-list li:hover .overlay {
+  opacity: 0.9;
+  height: 100%;
+}
```
Comment (ilya-ischenko) [inline homeworks/agustin-balbuena/04-css/styles/layout.css:202]: When transitioning height, the DOM has to re-flow (or re-layout) all affected elements. This is computationally expensive considering transitions attempt to animate at 60fps. That's 60 recalculations per second!

With transform scale, you're only affecting the element's appearance. In other words, translation does not affect layout, as it's applied post-layout.

File: homeworks/agustin-balbuena/04-css/styles/layout.css [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  grid-column-gap: 20px;
+  grid-row-gap: 20px;
+}
+
+ul li {
+  position: relative;
+}
```
Comment (ilya-ischenko) [inline homeworks/agustin-balbuena/04-css/styles/layout.css:139]: Bad practice to use elements selector

**Comment by agustinbalbuena** [comment]
Hi @ilya-ischenko  I hope everything is ok!
I think I've done everything you mentioned. please check it out when you have some time.

File: homeworks/agustin-balbuena/04-css/styles/layout.css [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+  margin: 2rem 0 3rem 0;
+}
+
+.profile-btns a {
+  width: calc((100% - 100px) / 3 + 79px);
+  background-color: #ffffff;
+  border-radius: 10px;
+  padding: 20px 10px;
+  text-align: center;
+}
```
[the remark this answers is not in the document]
Comment (agustinbalbuena) [reply homeworks/agustin-balbuena/04-css/styles/layout.css]: I was trying to get the same width to both buttons but honestly not sure 😅.
Resolved now

**Review by ilya-ischenko — APPROVED** [review]
Well done!
Approved.

**Pull request merged**